python/TestHarness/schedulers/RunPBS.py

```diff
@@ -57,19 +60,22 @@ def hasTimedOutOrFailed(self, job_data):
                     job.setStatus(job.error, 'QSTAT')
                 return True
 
-            qstat_job_result = re.findall(r'Exit_status = (.*)', qstat_command_result)
+            job_result = job_meta.get('Exit_status', False)
+            job_output = job_meta.get('Output_Path', False)
```

Comment:
Shouldn't we die more horribly if we can't find those keys? That's indicative of PBS changing

Reply:
I need to spend time catching up with this PR... long break, forget where I left off, that sort of thing.

We don't want to die on 'Exit_status' not being there, because this key will not exist until the job begins (if its Queued, this key does not exist). I don't know about the Output_Path. I think that _always_ exists, so... yes to dying horribly on that one.

Reply:
Output_Path is always present, set by the qsub script launching the job. So I removed the .get method.